charts/generic-prometheus-alerts/templates/_helpers.tpl

```diff
@@ -50,3 +51,22 @@ Selector labels
 app.kubernetes.io/name: {{ include "generic-prometheus-alerts.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
+
+{{/*
+Prometheus Rule labels
+*/}}
+{{- define "generic-prometheus-alerts.ruleLabels" -}}
+{{- $targetApplication := required "A value for targetApplication must be set" .Values.targetApplication }}
+{{- $businessUnit := required "A value for businessUnit must be set" .Values.businessUnit }}
+{{- $alertSeverity := required "A value for alertSeverity must be set" .Values.alertSeverity }}
+{{- $environment := default "null" (.Values.global).environment -}}
+{{- $productId := default "null" (.Values.global).productId -}}
```

Comment:
is there a reason why these aren't `required`?

Thinking if there's a breaking change then we might as well make people add them?

Reply:
There is a requirement to also upgrade to using the latest shared hmpps github actions - otherwise these wont get set. So by not making them required it allows for the old github actions workflows to work still, but perhaps that would be a good motivation to update both.